docs/themes/mokksy/assets/css/layout.css

```
@@ -57,6 +57,147 @@
+}
+
+.nav-toggle:hover {
+  border-color: var(--border-hover);
+  color: var(--text);
+}
+
+.nav-toggle span {
+  display: block;
+  width: 16px;
+  height: 1.5px;
+  background: currentColor;
```

Stylelint (17.11.1)

[error] Expected "currentColor" to be "currentcolor" (value-keyword-case)

(value-keyword-case)